lisa/microsoft/testsuites/cloud_hypervisor/ch_tests_tool.py

```diff
@@ -83,6 +84,36 @@ class CloudHypervisorTests(Tool):
     cmd_path: PurePath
     repo_root: PurePath
 
+    # Perf-stable profile configuration Set to "" to disable
+    perf_profile: str = "perf-stable"
```

Comment:
is the perf stable a term defined somewhere, or it's used here only? What does it mean?